frontends/web/src/locales/en/app.json

```diff
@@ -197,6 +197,40 @@
   "bitboxBase": {
     "connect": "Connect Base",
     "detectedBases": "Detected the following devices on your network:",
+    "errors": {
+      "BACKUP_SYSCONFIG_NOT_A_MOUNTPOINT": "Internal error: {{error}}",
+      "CHANGEPASSWORD_TOO_SHORT": "Password too short. A password must be at least 8 characters.",
+      "CMD_SCRIPT_INVALID_ARG": "Internal error: {{error}}",
+      "CONFIG_SCRIPT_INVALID_ARG": "Internal error: {{error}}",
+      "DUMMY_AUTHENTICATION_NOT_SUCCESSFUL": "Internal error: {{error}}",
+      "DUMMY_CHANGEPASSWORD_TOO_SHORT": "Internal error: {{error}}",
+      "ENABLE_CLEARNETIBD_TOR_ALREADY_DISABLED": "Tor is disabled. Cannot set Bitcoin Core to use Tor with public internet blockchain download.",
+      "EXECUTABLE_NOT_FOUND": "Internal error: {{error}}",
+      "FLASHDRIVE_CHECK_MULTI": "More than one flashdrive plugged in to the BitBoxBase, please remove all but one.",
+      "FLASHDRIVE_CHECK_NONE": "No flashdrive plugged in to the BitBoxBase, please insert one.",
+      "FLASHDRIVE_MOUNT_NOT_FOUND": "Internal error: {{error}}",
+      "FLASHDRIVE_MOUNT_NOT_SUPPORTED": "Flashdrive plugged in to the BitBoxBase is not supported (either too big or filesystem not supported).",
+      "FLASHDRIVE_MOUNT_NOT_UNIQUE": "Internal error: {{error}}",
+      "FLASHDRIVE_UNMOUNT_NOT_MOUNTED": "Internal error: {{error}}",
+      "MENDER_UPDATE_ALREADY_IN_PROGRESS": "Cannot update firmware, update is already in progress.",
+      "MENDER_UPDATE_COMMIT_FAILED": "Internal error: {{error}}",
+      "MENDER_UPDATE_IMAGE_NOT_MENDER_ENABLED": "Cannot update firmware because the disk image has been built without that functionality.",
+      "MENDER_UPDATE_INSTALL_FAILED": "Firmware installation to passive partition failed.",
+      "MENDER_UPDATE_INVALID_VERSION": "Internal error: {{error}}",
+      "MENDER_UPDATE_NO_VERSION": "Internal error: {{error}}",
+      "PROMETHEUS_ERROR": "Internal error: {{error}}",
+      "REDIS_ERROR": "Internal error: {{error}}",
+      "RESTORE_SYSCONFIG_BACKUP_NOT_FOUND": "No backup file detected on the flashdrive plugged in to the BitBoxBase.",
+      "SCRIPT_INCLUDES_NOT_FOUND": "Internal error: {{error}}",
+      "SCRIPT_NOT_FOUND": "Internal error: {{error}}",
+      "SCRIPT_NOT_RUN_AS_SUPERUSER": "Internal error: {{error}}",
+      "SET_BITCOINDBCACHE_INVALID_VALUE": "Internal error: {{error}}",
+      "SET_BITCOINETWORK_INVALID_VALUE": "Internal error: {{error}}",
+      "SET_HOSTNAME_INVALID_VALUE": "Proposed name for BitBoxBase is not valid.",
+      "SET_NEEDS_TWO_ARGUMENTS": "Internal error: {{error}}",
+      "SET_ROOTPASSWORD_PASSWORD_TOO_SHORT": "Password too short. A password must be at least 8 characters.",
```

Comment:
Should remember to rename things here as well once we do it...

Reply:
Worth adding a comment to the `errorcodes.go` file in both repos to remember to update `app.json` when making changes

Reply:
The problem is that we shouldn't change anything here after pushing to locize. Those changes then need to be made upstream in locize and then pulled into here. So I would keep this PR open for now until the release candidate when the error codes are more finalized